### DISCUSSION TITLE
@swc/core threw an error

### DISCUSSION THREAD

**Discussion by budok42** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#341] [Q&A] [answered]
Pouvez-vous m'aider à résoudre ce problème ?
![2024-08-21_16-00-22](https://github.com/user-attachments/assets/8750de6b-7b81-4545-a185-0e910fc0d433)

**Comment by Laikker** [comment, marked as answer]
Hello, change le niveau de swc-core comme indiqué ici:
https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/discussions/321#discussioncomment-9981024

**Reply by budok42** [reply]
Bonjour, Je m'excuse de vous importuner, mais je ne suis qu'un débutant... Pourriez-vous m'expliquer comment faire (ou me rediriger vers une page d'instructions ? Merci d'avance.)

**Reply by Laikker** [reply]
Tu comprends pas quoi? il faut juste modifier le fichier indiqué avec la valeur que j'ai mise

**Reply by budok42** [reply]
J'ai apporté des modifications à ce fichier, mais l'erreur persiste : .... Peut-être que je fais quelque chose de mal ? 
[package.json](https://github.com/user-attachments/files/16692201/package.json)

**Reply by Laikker** [reply]
tu a refait un yarn install? et un yarn build?

**Reply by budok42** [reply]
Ohhhh no.... Nécessité de reconstruire le core et phone ?)

**Reply by Laikker** [reply]
juste le core

**Reply by budok42** [reply]
Merci beaucoup pour votre aide rapide et efficace.
